module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Remove-EntraUser.md

```diff
@@ -26,7 +26,7 @@ Removes a user.
 
 ```powershell
 Remove-EntraUser 
- -ObjectId <String>
+ -UserId <String>
  [<CommonParameters>]
 ```
 
```

Comment:
Update the description section which is still referencing the ObjectId. Also, add backticks to the command name `Remove-EntraUser`